skrf/calibration/calibration.py

```diff
@@ -2350,6 +2350,12 @@ def __init__(self, measured, ideals=None, estimate_line=False,
 
         The reflect ideals can also be given as a +-1.
 
+        If thru is non-zero length, the calibration is done with zero length thru and
+        the ideal thru length is subtracted from the ideal lines. The resulting
+        calibration reference plane is at the center of the thru. If reflects
+        are passed as networks the correct reference plane is center of the
```

Comment:
Maybe "If reflects are passed as networks instead of None or float"? I had to read twice to understand they can be passed with other types. But it makes sense if the full docstring is read and not only the diff.